app/posts/book-a-vaccination/2025/05/2025-05-22-improving-location-search.md

```diff
@@ -0,0 +1,102 @@
+
+ 
+
+**Option 1 – Search and return results on next page** 
+
+We designed a version that was similar to the service search function on NHS.uk.  This allowed the user to enter either: 
+
+- full postcodes
+- partial postcodes
+- city, town, or village names 
+
+If the user enters a full or partial postcode, the search returns a list of results based on the geographical centre of the information that was entered. 
+
+If the user enters a city or town name, the system returns a list of potential matches for the user to select, before showing a list of results in the chosen location. 
+
+![screenshot showing a set of screens detailing the enhanced search function](enhancedsearch.png) 
+
+ 
+
+ 
+
+**Option 2 – Enhanced autocomplete search** 
+
+Another option we considered was to use an autocomplete component to immediately return results based on the user input.  This would still allow users to search using full or partial postcodes, and town names. 
+
+![screenshot showing a set of screens detailing the autocomplete search function](autocomplete-search.png) 
+
+After discussing this option with the team, it was decided that the amount of data that would need to be queried using this functionality would affect the service performance too much. 
+
+## What we tested 
+
+We decided to test the first design option.  We also made some enhancements to the results page, to help reduce confusion on this page. 
+
+**Enhanced search page** 
+
+![screenshot showing the enhanced search input](enhanced-search-input.png) 
+
+**Location selection page** 
+
+![screenshot showing a list of potential locations for a user to choose from](town-results.png) 
+
+**Site results page** 
+
+We updated the site list page to reflect how sites are displayed on NHS.uk, and added the number of sites near the top of the page. 
+
+![screenshot showing a list of available vaccination sites for a user to choose](newsitelist.png) 
+
+ 
+
+## Findings 
+
+Most participants preferred using a full postcode for accuracy, especially those living in larger cities like London. Two participants said they would prefer to enter a town name. 
+
+**Location selection** 
+
+Participants said the list of possible locations appeared clear and that they would be happy to select from it. 
+
+The ability to try another town, city or postcode was seen as a helpful fallback for incorrect or unexpected matches by one participant. 
+
+> Entering the town name is good, I like it, quite clever because if my dad is visiting a different town we can get a site there for his jab. 
+
+**Choosing a site**
+
+Most users found the information clear and said provided all the details they needed.  Participants generally preferred choosing the nearest site or one they had visited before.  
+
+## Future considerations
+
+We may consider adding the 'use your location' function from the 'find a' service on NHS.uk in future.  
```

Comment:
I'm not sure that 'NHS.uk' is the right way to refer to the website? They seem to have moved away from that terminology?

```suggestion
We may consider adding the 'use your location' function from the 'find a' service on the NHS website in future.  
```